packages/gatsby/src/builder-config.js

```diff
@@ -30,6 +31,6 @@ module.exports.getGQLOptions = options => {
     // Field under which the remote schema will be accessible. You'll use this in your Gatsby query
     fieldName: config.fieldName,
     // Url to query from 30
-    url: `${config.baseURL}/${config.publicAPIKey}?cachebust=true`,
+    url: `${config.baseURL}/${config.publicAPIKey}?${!config.useCache ? "cachebust=true" : ''}`,
```

Comment:
@teleaziz what's the best process/setup to test changes to this plugin? `npm link` to a gatsby starter and point to local API?